lib/db.js

```
@@ -468,6 +468,12 @@ DB.prototype._put = function(req) {
     }
     var schema = req.schema;
     var query = req.query;
+    var missingColumn = Object.keys(query.attributes).find(function(attrName) {
```

We use buildPutQuery for both the indexes [and the main data update](https://github.com/wikimedia/restbase-mod-table-cassandra/blob/master/lib/db.js#L498). We also [already check whether attributes are in the schema](https://github.com/wikimedia/restbase-mod-table-cassandra/blob/master/lib/dbutils.js#L693-L705). I think the main bit that's missing is to throw an error, rather than silently dropping extra attributes.